BUG: software version is not mandatory in CCA tokens

Trying to extract reference values from a token whose software components are not explicitly versioned, I get:

```sh
09:52 $ gen-corim cca cca_example_token.cbor cpak-pub.json --template-dir corim-templates
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x338184]

goroutine 1 [running]:
github.com/veraison/gen-corim/cmd.GetMeasurementsFromComponents({0x40000b0fc8?, 0x4000098500?, 0xffffc93eb201?}, {0x40000a763c, 0x4, 0x4}, 0x1)
	/home/tho/go/pkg/mod/github.com/veraison/gen-corim@v0.0.0-20231004131939-f515bfe3ad9a/cmd/gen-corim.go:267 +0x154
github.com/veraison/gen-corim/cmd.generate(0x400016cfa0, 0x400016cfb0, 0x400016cfc0, 0x4000110430, 0x4000110440)
	/home/tho/go/pkg/mod/github.com/veraison/gen-corim@v0.0.0-20231004131939-f515bfe3ad9a/cmd/gen-corim.go:144 +0x260
github.com/veraison/gen-corim/cmd.NewRootCmd.func1(0x400018a100?, {0x400016cfa0?, 0x7?, 0x3e26be?})
	/home/tho/go/pkg/mod/github.com/veraison/gen-corim@v0.0.0-20231004131939-f515bfe3ad9a/cmd/gen-corim.go:69 +0xe8
github.com/spf13/cobra.(*Command).execute(0x40001b6008, {0x4000122130, 0x5, 0x5})
	/home/tho/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x654
github.com/spf13/cobra.(*Command).ExecuteC(0x40001b6008)
	/home/tho/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
	/home/tho/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/veraison/gen-corim/cmd.Execute()
	/home/tho/go/pkg/mod/github.com/veraison/gen-corim@v0.0.0-20231004131939-f515bfe3ad9a/cmd/gen-corim.go:109 +0x24
main.main()
	/home/tho/go/pkg/mod/github.com/veraison/gen-corim@v0.0.0-20231004131939-f515bfe3ad9a/main.go:11 +0x1c
```

Since the version claim is not mandatory, the program should not break.

--- a/cmd/gen-corim.go
+++ b/cmd/gen-corim.go
@@ -263,8 +263,12 @@ func GetMeasurementsFromComponents(swComponents []psatoken.SwComponent, config [
 
 	for _, component := range swComponents {
 		refValID := comid.NewPSARefValID(*component.SignerID)
-		refValID.SetLabel(*component.MeasurementType)
-		refValID.SetVersion(*component.Version)
+		if component.MeasurementType != nil {
+			refValID.SetLabel(*component.MeasurementType)
+		}
+		if component.Version != nil {
+			refValID.SetVersion(*component.Version)
+		}
 		measurement := comid.NewPSAMeasurement(*refValID)
 		measurement.AddDigest(1, *component.MeasurementValue)
 		measurements.AddMeasurement(measurement)

